File: src/main/java/com/acme/bnb/controlers/clases/DashboardLessorData.java
package com.acme.bnb.controlers.clases;

import java.util.List;
import lombok.Data;

@Data
public class DashboardLessorData {
    List<PropertyWithNumber> propertiesWithAuditCount;
    List<PropertyWithNumber> propertiesWithRequestCount, propertiesWithApprovedRequestCount, propertiesWithDeniedRequestCount, propertiesWithPendingRequestCount;
}

File: webapp/src/app/service/class/dashboard-lessor-data.ts
import { PropertyWithNumber } from "./property-with-number";

export class DashboardLessorData {
    propertiesWithAuditCount: PropertyWithNumber[] = [];
    propertiesWithRequestCount: PropertyWithNumber[] = [];
    propertiesWithApprovedRequestCount: PropertyWithNumber[] = [];
    propertiesWithDeniedRequestCount: PropertyWithNumber[] = [];
    propertiesWithPendingRequestCount: PropertyWithNumber[] = [];
}
